ubuntu20to22/upgrader.py

```diff
@@ -112,6 +112,10 @@ def construct_actions(
                 #  we support following PHP versions: PHP 7.1, 7.2, 7.3.
                 actions.UpdateLegacyPhpRepositories(self._distro_from, self._distro_to),
                 actions.SetupUbuntuRepositories("focal", "jammy"),
+                actions.ReplaceAptReposRegexp(
+                    r'(http|https)://([^/]+)/(.*\b)20\.04(\b.*)',
```

Comment:
This looks scary :laughing: 
But ok, let's try this approach

Reply:
For me replacing strings naively is more scary :)
I expect, at most no-op (transform the string as is) when it could harm.